velox/exec/tests/DriverTest.cpp

```diff
@@ -267,10 +267,11 @@ class DriverTest : public OperatorTestBase {
     return task;
   }
 
-  std::shared_ptr<Task> testDriverSuspensionWithTaskOperationRace(
+  void testDriverSuspensionWithTaskOperationRace(
       int numDrivers,
       StopReason expectedEnterSuspensionStopReason,
       StopReason expectedLeaveSuspensionStopReason,
+      TaskState expectedTaskFinishedState,
```

Comment:
Looks like only 2 states are supported: Finished and Aborted. Should we use a boolean instead?

Reply:
@mbasmanova thanks for the quick review. Updated!